Networking/Networking/Mapper/ProductCategoryMapper.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+/// Mapper: ProductCateogy
+///
+struct ProductCategoryMapper: Mapper {
```

Comment:
This doesn't seem to be used anywhere. I guess we would be using it later? It's also missing tests but we can do that later. 🙂

Reply:
Yeah, I'll delete it and add it later if needed!